_i18n/de/resources/moneropedia/consensus.md

```diff
@@ -1,9 +1,9 @@
 ---
-terms: ["consensus", "consensus-network"]
-summary: "consensus describes a property of distributed networks like Monero where most of the participants follow the rules, and thus reject bad participants"
+terms: ["consensus", "consensus-network", "Konsens", "Konsens-Netzwerk"]
+summary: "Beschreibt eine Eigenschaft dezentralisierter Netzwerke wie Monero, in denen der Großteil der Mitwirkenden die bestehenden Regeln befolgt und dementsprechend schädliche Beteiligte abgwiesen werden"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-Consensus describes a property of distributed networks like Monero where most of the participants follow the rules, and thus reject bad participants.
\ No newline at end of file
+Konsens beschreibt eine Eigenschaft dezentralisierter Netzwerke wie Monero, in denen der Großteil der Mitwirkenden die bestehenden Regeln befolgt und dementsprechend schädliche Beteiligte abgwiesen werden.
```

Comment:
```suggestion
Konsensus beschreibt eine Eigenschaft verteilter Netzwerke wie Monero, in denen der Großteil der Mitwirkenden die Regeln befolgt und dementsprechend schädliche Beteiligte abgwiesen werden.
```

Reply:
Changed, except "Konsensus". What would be the substancial difference to "Konsens", @stmax82 ? If "Konsens" is technically incorrect, I'll also change this in the translation of the website via weblate.

Reply:
Konsens is fine. I thought Konsensus is used more in this context, but I couldn't confirm. Both are rather rarely used in German :)